src/app/rpmostree-compose-builtin-tree.cxx

```diff
@@ -1661,3 +1661,93 @@ rpmostree_compose_builtin_extensions (int             argc,
 
   return TRUE;
 }
+
+static void
+state_action_changed_cb (DnfState       *state,
```

Comment:
Hmm...I think we could actually just do this by default in `rpmostree-core.cxx` right?  I believe we bypass this section of libdnf in the ostree-native case.  That way it'd work out of the box in other code paths too.